CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ Follow either of the two links above to access the appropriate CLA and instructi
 
 ## How to Build and Test
 
-Look into `./test.sh` to understand how. Minimally,
+Minimally,
```

Comment:
Why not keeping the script? I think it's will be a hassle for me to open `CONTRIBUTING.md` and copy-paste the following commands every time I come back to this repo to do some work. (I always forget how to build.)
```
bazel build --curses=no //package_manager:dpkg_parser.par	
bazel build --curses=no //...	
bazel test --curses=no --test_output=errors --test_timeout=900 //...	
```
```
buildifier -mode=fix $(find . -name 'BUILD*' -o -name 'WORKSPACE*' -o -name '*.bzl' -type f)
```
```
find . -name "*.py" | xargs pylint --disable=R,C
```
It's a lot of commands.

Reply:
I brought it back, and restored the line I removed, since this seems optimized for local dev.

Generally my bias is towards `go test ./...` or `bazel test //...` just working in repos to avoid the need for bespoke scripts, but we're clearly not there (just shining a light into where my bias comes from).